### PR TITLE
TSK-360: Assignee selection enhancements

### PR DESCRIPTION
**Pull request by disant1**
Signed-off-by: Denis Maslennikov <denis.maslennikov@gmail.com>

# Contribution checklist

## Brief description
Screenshot 2023-01-16 at 20 56 01

Screenshot 2023-01-16 at 20 56 25

Screenshot 2023-01-16 at 20 56 33

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.

* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

**Comment by haiodo** [comment]
@disant1 Please add screenshots for UI changes if possible. Thanks.

File: packages/presentation/package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "@hcengineering/login": "^0.6.1",
     "@hcengineering/image-cropper": "^0.6.0",
     "@hcengineering/client": "^0.6.6",
+    "@hcengineering/tracker": "^0.6.1",
```
Comment (haiodo) [inline packages/presentation/package.json]: This is not allowed, presentation is generic package, it should not know about tracker.

File: packages/presentation/src/components/AssigneeBox.svelte [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import contact, { Contact, Employee, formatName } from '@hcengineering/contact'
+  import { AttachedData, Class, DocumentQuery, FindOptions, Ref } from '@hcengineering/core'
+  import { getEmbeddedLabel, IntlString } from '@hcengineering/platform'
+  import { Issue, IssueTemplateData } from '@hcengineering/tracker'
```
Comment (haiodo) [inline packages/presentation/src/components/AssigneeBox.svelte]: Please do generic component, based on field name specification.

**Review by haiodo — CHANGES_REQUESTED** [review]

File: packages/presentation/package.json (hunk 1 above)
Comment (disant1) [reply packages/presentation/package.json]: Fixed

File: packages/presentation/src/components/AssigneeBox.svelte (hunk 2 above)
Comment (disant1) [reply packages/presentation/src/components/AssigneeBox.svelte]: Done

**Review by haiodo — APPROVED** [review]

**Pull request merged**